src/core/message.rs

```diff
@@ -807,4 +834,4 @@ impl WriteBytes for ControlChangeValue {
         write_u8!(w, self.value.get())?;
         Ok(())
     }
-}
+}
```

Comment:
Can you run `cargo fmt` and force push the fix to this commit?

Reply:
@webern - I think I did this but hopefully got the commit/push right. Let me know if not.

Reply:
The issue was changes to the clippy linter. I have pushed a fix as a separate commit to this PR.